### PR TITLE
fixup formatting to use only tabs

### PR DESCRIPTION
**Pull request by jlhogan**
also ensure backend formatting runs over test code as well
(build failure is just a warning, and does not break to formatting)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -57,7 +57,7 @@ dotnet-format:
 	dotnet format
 
 dc-dotnet-format:
-	./dc-backend dotnet format
```
Comment (cailyncodes) [inline Makefile]: This old approach should work?

We should also probably use the `./dc` helper if we are changing it though

Comment (jlhogan) [reply Makefile]: the `working_dir` for the backend container is `app/src/Hedwig`; we need to run this commend at the root `/app` level to get it to apply to all files. 

updating to use `.dc` helper tho!

Comment (cailyncodes) [reply Makefile]: We can also probably fix the docker-compose file?

Comment (jlhogan) [reply Makefile]: what should be fixed in docker-compose file?

**Pull request merged**